content/ytt/docs/latest/ytt-schema.md

```diff
@@ -0,0 +1,60 @@
+---
+title: ytt @schema/match
+---
+
+## Overview
+
+`ytt` schemas are currently in the **experimental** phase. To use schema features, include `--enable-experiment-schema`.
+
+Configuration authors use Schema to verify data value YAML structures and their values and define default data value configuration.
+
+Using a schema guarantees to templates that all data values exist and are of the correct type. This alleviates templates from doing existence and type checks themselves.
+
+Common use cases:
+
+1. Set default values
+1. Provide visibility to valid values that could be updated by the consumer 
+1. Provide visibility to valid types and YAML structure that could be updated by the consumer
+
+Consider this simple example schema provided by the _configuration author_:
+
+> `schema.yml`
+> ```yaml
+> #@schema/match data_values=True
+> ---
+> load_balancer:
+>   enabled: true
+>   external_ip: ""
+> ```
+A configuration _consumer_ may add their own customized data values. The data values used are limited to only the values in the schema above, and if supplied, `load_balancer.enabled` must be a `bool`, and `load_balancer.external_ip` must be a `string`.
+
+> `values.yml`
+> ```yaml
+> #@data/values
+> ---
+> load_balancer:
+>   external_ip: 172.120.12.232
+> ```
+
+Then paired with a template
+> `config.yml`
+> ```yaml
+> #@load("ytt:data", "data")
+> ---
+> service: #@ data.values.load_balancer
+> ```
+> 
```

Comment:
If you pair-up the template with the schema, it hits home that the Configuration Author writes both of _those_ files... and then in the next scene, the Configuration Consumer writes the `values.yml`.

Reply:
Great point